### DISCUSSION TITLE
Use with SQLite

### DISCUSSION THREAD

**Discussion by rico** [brefphp/bref#1497] [Support]
Does anybody know a way to use an SQLite database with this (bref / AWS)? Thanks!

**Comment by afu-dev** [comment]
Hello Rico, you can use Amazon EFS to create a filesystem and mount it on your lambdas. Then store the SQLite file in it.

Some reading I could found: https://www.lambrospetrou.com/articles/aws-lambda-and-sqlite-over-efs/

**Comment by rico** [comment]
Thanks @afu-dev will look into this.
